examples/combined-hbs-jsx/views/home.hbs

```diff
@@ -0,0 +1,10 @@
+<div style="background-color:#FFF1EF; padding:20px">
+  <h1>{{pageTitle}}</h1>
+  <p>This page has been rendered using Handlebars as the view engine</p>
+
+  {{#capture "myCapturedPartial"}}
+    {{> partial title="This is a Handlebars partial"}}
+  {{/capture}}
+
+  {{{loadJsxComponent partial=myCapturedPartial}}}
```

Comment:
The property and variable name may imply this functionality only works for partial templates, perhaps renaming `partial` to `html` and `myCapturedPartial` to `myCapturedOutput` or similar would avoid any potential confusion?

Reply:
Fixed in this [commit](https://github.com/Financial-Times/dotcom-page-kit/pull/714/commits/0d1bd1e93c86a7251ed186aa691fcfb03872e479).